fem/pfespace.cpp

```diff
@@ -3541,7 +3541,6 @@ DeviceConformingProlongationOperator::DeviceConformingProlongationOperator(
    const int tdofs = R->Height();
    MFEM_ASSERT(tdofs == R->HostReadI()[tdofs], "");
    ltdof_ldof = Array<int>(const_cast<int*>(R->HostReadJ()), tdofs);
-   ltdof_ldof.UseDevice();
```

Comment:
This is a no-op. More generally, `UseDevice()` is a bit of a footgun, it's so similar to `UseDevice(true)`